.github/workflows/README.md

```diff
@@ -78,10 +75,13 @@ To run a full build and download all datasets, you would then combine both tags:
 $ git commit -m "[full doc][force download] request full build"
 ```
 
-#### Hosting of the docs
+## Building the stable release documentation
 
-Upon a successful build of the doc, it is zipped and uploaded as an artifact.
-A circle-ci workflow is then triggered. See below.
+### release-docs.yml
+
+Should be triggered automatically after merging and tagging a release PR to
```

Comment:
We should make a note to check that on the next release. Because I think we had to run it manually last time.

Reply:
Yeah to be fair, we'll know if it works at the next release. :-p
Though as it is a manual workflow we could also test it now.